src/main/resources/assets/farmersdelight/lang/it_it.json

```diff
@@ -31,28 +31,28 @@
   "block.farmersdelight.rice_bag": "Sacco di riso",
   "block.farmersdelight.straw_bale": "Balla di paglia",
 
-  "block.farmersdelight.canvas_sign": "Cartello di tela",
-  "block.farmersdelight.white_canvas_sign": "Cartello di tela bianco",
-  "block.farmersdelight.orange_canvas_sign": "Cartello di tela arancione",
-  "block.farmersdelight.magenta_canvas_sign": "Cartello di tela magenta",
-  "block.farmersdelight.light_blue_canvas_sign": "Cartello di tela azzurro",
-  "block.farmersdelight.yellow_canvas_sign": "Cartello di tela giallo",
-  "block.farmersdelight.lime_canvas_sign": "Cartello di tela lime",
-  "block.farmersdelight.pink_canvas_sign": "Cartello di tela rosa",
-  "block.farmersdelight.gray_canvas_sign": "Cartello di tela grigio",
-  "block.farmersdelight.light_gray_canvas_sign": "Cartello di tela grigio chiaro",
-  "block.farmersdelight.cyan_canvas_sign": "Cartello di tela ciano",
-  "block.farmersdelight.purple_canvas_sign": "Cartello di tela viola",
-  "block.farmersdelight.blue_canvas_sign": "Cartello di tela blu",
-  "block.farmersdelight.brown_canvas_sign": "Cartello di tela marrone",
-  "block.farmersdelight.green_canvas_sign": "Cartello di tela verde",
-  "block.farmersdelight.red_canvas_sign": "Cartello di tela rosso",
-  "block.farmersdelight.black_canvas_sign": "Cartello di tela nero",
-
+  "block.farmersdelight.canvas_sign": "Cartello di Tela",

```

Comment:
This is an optional suggestion, we can merge either way.

Some languages (such as `pt_br`) use a "Sentence-case capitalization", where only the first word stars so, as opposed to the "Title Case Capitalization" which `en_us` uses. If the official Italian lang uses sentence-case, it might be interesting to preserve that across lines for consistency.

Reply:
Otherwise, the lang looks good! Let me know if this is an interesting change, or if I can proceed with the merge. (y)

Reply:
Usually `it_it` use _Sentence-case capitalization_ i just used **en** capitalization as reference. Great consideration!
If u wish to use a right italian language is just  _Sentence-case capitalization_ otherwise u can use some rule like [_"Chicago Manual of Style"_](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style) because we're talking of items names and not sentences, but to be honest is more a context problem and can be really complex, i know italian very well but i'm not a language expert in this way .-.

Reply:
> Usually it_it use Sentence-case capitalization i just used en capitalization as reference. Great consideration!

I see! Then it might be more consistent to change them to sentence-case. Though I leave that up to you; just give me a heads-up when I can merge.